php-nginx/composer.sh

```diff
@@ -23,10 +23,10 @@ DEFAULT_PHP_VERSION="5.6"
 if [ -f ${APP_DIR}/composer.json ]; then
     # Extract php version from the composer.json.
     CMD="${PHP_DIR}/bin/php /tmp/detect_php_version.php ${APP_DIR}/composer.json"
-    PHP_VERSION=`${CMD}`
+    PHP_VERSION=`su www-data -c "${CMD}"`
 
     # Remove the vendor directory for the temporary script above.
-    rm -rf /tmp/vendor
+    rm -rf /tmp/vendor /tmp/detect_php_version.php /tmp/composer.*
```

Comment:
Ok. Can we update the comment to reflect that? `# Remove files and directories created in Dockerfile` or something along those lines